test-cases/manager/manager-repair-intensity-multiple-repaired-nodes.yaml

```diff
@@ -0,0 +1,32 @@
+test_duration: 1200
+
+prepare_write_cmd: [
+    "cassandra-stress write cl=QUORUM n=73242192 -schema 'replication(factor=3)' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..73242192 -port jmx=6868 -mode cql3 native -rate threads=200 -log interval=5",
+    "cassandra-stress write cl=QUORUM n=73242192 -schema 'replication(factor=3)' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=73242193..146484384 -port jmx=6868 -mode cql3 native -rate threads=200 -log interval=5",
+    "cassandra-stress write cl=QUORUM n=73242192 -schema 'replication(factor=3)' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=146484385..219726576 -port jmx=6868 -mode cql3 native -rate threads=200 -log interval=5",
+    "cassandra-stress write cl=QUORUM n=73242192 -schema 'replication(factor=3)' -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=219726577..292968768 -port jmx=6868 -mode cql3 native -rate threads=200 -log interval=5",
+]
+stress_read_cmd: "cassandra-stress read cl=QUORUM duration=1020m -schema 'replication(factor=3)' -col 'size=FIXED(1024) n=FIXED(1)' -port jmx=6868 -mode cql3 native -rate threads=125 throttle=<THROTTLE_PLACE_HOLDER>/s -log interval=5"
+round_robin: true
+
+hinted_handoff: 'disabled'
```

Comment:
maybe a small comment, as for why we disable hinted handoff here

Reply:
Because when we run the `create_fault` function, whose purpose is to run C-S in such a way that every node will have missing rows, hinted handoff would simply circumvent this effort, and as a result there would be no missing rows to repair.

Reply:
I could guess this, but it would be nice to have this comment in the YAML